docs/advanced/advanced_examples.rst

```diff
@@ -0,0 +1,27 @@
+.. _advanced-advanced_examples-label:
+
+=================
+Advanced examples
+=================
+
+Setting style
+=============
+
+The default style of ``plothist`` is presentation-ready, with large and readable axis labels and legend. To have a paper-ready plotting style, you can use:
```

Comment:
I would explicitly say 

* `import plotthist` automatically sets the matplotlib style to a presentation-ready style.

And then start a new paragraph for the publication-ready (not paper-ready, too jargonic)